TrustModel_Personas.md

```diff
@@ -0,0 +1,195 @@
+  controlled by a Key Broker Service).
+- Provides other required verification artifacts (secure channel may be considered).
+- Protects the keys used to sign or encrypt the container images.
+
+It is recognised that hybrid options exist surrounding workload provider and container provider. 
+For example the workload provider may choose to protect their supply chain by 
+signing/encrypting their own container images after following the build patterns already 
+established by the container image provider. 
+
+Example : Sidecar Container Provider
+
+
+### Data Owner
+Owner of data used, and manipulated by the application.
+- Concerned with visibility and integrity of their data.
+- Concerned with compliance and protection of their data.
+- Uses and shares data with solutions. 
+- Wishes to ensure no visibility or manipulation of data is possible by 
+  Orchestration Operator or Cloud Operator personas.
+
+## Conclusion
+
+The key trust relationship here is between the Data Owner and the other personas. The Data Owner 
+trusts the code in the form of container images chosen by the Workload Provider to operate across 
+their data, however they do not trust the Orchestration Operator or Cloud Operator with their 
+data and wish to ensure data confidentiality.
+
+The Workload Provider must not trust the Orchestration Operator or Cloud Operator otherwise 
+they risk losing the trust of the Data Owner. This means the Workload Provider must
+provide assurance to the data owner regarding the integrity of the code and that the data is 
+appropriately protected.
+
+The Workload Provider is free to choose Container Image Providers that will provide not only 
+the images they need but also support the verification method they require.  A key aspect to this
+relationship is the Workload Provider applying Supply Chain 
+Security practices (as 
+described on Page 42 of
+[Cloud Native Security Paper](https://github.com/cncf/tag-security/blob/main/security-whitepaper/v2/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf)
+) when considering Container 
+Image Providers. So the Container Image Provider must support the Workload Providers 
+ability to provide assurance to the Data Owner regarding integrity of the code.
+
+With Confidential Containers we match the TEE boundary to the most restrictive boundary which is 
+between the Workload Provider and the Orchestration Operator. 
+
+Outside the TEE we distinguish between the Orchestration Operator and the Infrastructure 
+Operator due to nature of how they can impact the TEE and the concerns of Workload Provider and 
+Data Owner. Direct threats exist from the Orchestration Operator as some orchestration actions  
+must be permitted to cross the TEE boundary otherwise orchestration cannot occur. A key goal is to 
+*deprivilege orchestration* and put a persona inside the TEE boundary in control of the 
+orchestration operators privileges across the boundary. However indirect threats exist 
```

Comment:
```suggestion
must be permitted to cross the TEE boundary otherwise orchestration cannot occur. A key goal is to 
*deprivilege orchestration* and restrict the 
Orchestration Operators privileges across the boundary. However indirect threats exist 
```
We don't really have a separate persona for this so I removed that text.